### PR TITLE
NVDA Example

### PR DESCRIPTION
**Pull request by mgifford**
Along with an error, I can't work through.

```
$ npx ts-node src/opat.ts validate -f opat/NVDA-2018.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml
Invalid: catalog file is not in YAML format
```

### CODE REVIEW

File: opat/NVDA-2018.yaml [hunk 1]
```diff
@@ -0,0 +1,1364 @@
...
+  company_name: NV Access
+  address: PO Box 4326, St Lucia South, QLD 4067, Australia
+  email: info@nvaccess.org
+  website: https://www.nvaccess.org/
+notes: ""
+evaluation_methods_used: "Vendor product knowledge"
+legal_disclaimer: ""
+chapters:
+  success_criteria_level_a:
+    notes: "GovReady-Q is a web application and an authoring tool. As such the Software & Authoring Tool Notes are all included within the Web for the 'Conformance Level' and 'Remarks and Explanations'. There are some elements here that clearly fail WCAG 2.0 A."
```
Comment (dmundra) [inline opat/NVDA-2018.yaml:17]: @mgifford this note says 'GovReady'.

Comment (dmundra) [reply opat/NVDA-2018.yaml:17]: Looks like I missed that but it appears in Moodle and Plone as well.

File: opat/NVDA-2018.yaml [hunk 2]
```diff
@@ -0,0 +1,1364 @@
...
+          - name: "software"
+            adherence:
+              level: "not-applicable"
+          - name: "authoring-tool"
+            adherence:
+              level: "not-applicable"
+      - num: "2.4.1"
+        components:
+          - name: "web"
+          - name: "web"
```
Comment (dmundra) [inline opat/NVDA-2018.yaml:232]: @mgifford duplicate `name: "web"` here.

**Review by dmundra — CHANGES_REQUESTED** [review]
Found the issue and commented on it.

Next one you will run into is:
> Invalid: criteria '504.2' is not included in 'Chapter 5: Software', criteria '504.2.3' is not included in 'Chapter 5: Software', criteria '504.2.4' is not included in 'Chapter 5: Software', criteria '602.3' is not included in 'Chapter 6: Support Documentation and Services'

File: opat/NVDA-2018.yaml [hunk 3]
```diff
@@ -0,0 +1,1364 @@
...
+      - num: "2.4.8"
+        components:
+          - name: "web"
+            adherence:
+              level: "partially-supports"
+              notes: "On the web site breadcrumbs are not provided."
+          - name: "software"
+            adherence:
+              level: "supports"
+              notes: "In NVDA itself, there are no “breadcrumb trails” to follow."
```
Comment (dmundra) [inline opat/NVDA-2018.yaml:675]: @mgifford quotes in quotes I think are causing the pre-commit failures. If you switch this to regular quotes and escape them it should be fine.

**Comment by mgifford** [comment]
I think I have all of these changes now

**Comment by dmundra** [comment]
> I think I have all of these changes now

I get this validation error. Did I miss these in the catalog?

```
> npx ts-node src/opat.ts validate -f opat/NVDA-2018.yaml -c catalog/2.4-edition-wcag-2.0-508-en.yaml
Invalid: criteria '504.2' is not included in 'Chapter 5: Software', criteria '504.2.3' is not included in 'Chapter 5: Software', criteria '504.2.4' is not included in 'Chapter 5: Software', criteria '602.3' is not included in 'Chapter 6: Support Documentation and Services'
```

**Comment by dmundra** [comment]
@mgifford I pushed up a fix to the catalog and the NVDA YAML. It is validating now. Do you want to generate the HTML and Markdown?

**Comment by dmundra** [comment]
@mgifford added a fix to the error you were getting earlier. I am done with my changes so back to you.

**Comment by mgifford** [comment]
Ok, .html & .markdown are there. Let me know what you think @dmundra

**Review by dmundra — APPROVED** [review]
Looking good. Thank you!

**Pull request merged**